lib/docs/filters/react_bootstrap/clean_html.rb

```diff
@@ -0,0 +1,17 @@
+module Docs
+  class ReactBootstrap
+    class CleanHtmlFilter < Filter
+      def call
+        css('.flex-column.d-flex').remove
+        css('header').remove
+        css('.bs-example').remove
+        css('.position-relative').each do |node|
+          code = node.at_css('textarea')
+          code.name = 'pre'
+          code['style'] = code['style'] + '; border: solid 1px;'
```

Comment:
- [ ] The `.position-relative` does not seem very specific for a code fragment. What about using `css('pre').ancestors('.position-relative').first`?
- [ ] Please extract the raw JSX code from the code fragment and rely on DevDoc's Prism.js syntax highlighting. See other scrapers for some examples. Most of them follow a patter such as: 

```ruby
node.content = node.content
node['data-language'] = 'jsx'
```